roles/ceph-common/handlers/main.yml

```diff
@@ -2,41 +2,19 @@
 - name: update apt cache
   apt:
     update-cache: yes
+  when: ansible_os_family == 'Debian'
 
 - name: restart ceph mons
-  service:
-      name: ceph-mon@{{ monitor_name }}
-      state: restarted
-  when:
-    - socket.rc == 0
-    - mon_group_name in group_names
+  include: "{{ playbook_dir }}/roles/ceph-common/handlers/restart-mon.yml"
```

Comment:
You might be right, I don't remember why I use the full path. Oh maybe because I was having issue with Ansible and relative/full path...